sample/allofhands_hawaii2016/sessions

```diff
@@ -0,0 +1,3467 @@
+		"title": "Core Browser 2016 Review / 2017 H1 Planning",
+		"track": {
+			"id": 8,
+			"title": "Firefox"
+		},
+		"start_time": "2016-12-07T18:45:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-07T21:30:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 5,
+			"name": "king's Ballroom 3"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 56
+	},
+	{
+		"short_abstract": "",
+		"speakers": [],
+		"subtitle": null,
+		"language": "english",
+		"title": "SensorWeb: Team Kick-off cont'd.",
+		"track": {
+			"id": 1,
+			"title": "All Hands"
+		},
+		"start_time": "2016-12-07T19:00:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-07T20:00:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 14,
+			"name": "Kona Ballroom 5"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 58
+	},
+	{
+		"short_abstract": "",
+		"speakers": [{
+			"id": 28,
+			"name": "Jim Cook",
+			"organisation": "Mozilla"
+		}],
+		"subtitle": null,
+		"language": "english",
+		"title": "FP&A Session with Jim",
+		"track": {
+			"id": 6,
+			"title": "FP&A"
+		},
+		"start_time": "2016-12-07T19:00:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-07T20:00:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 11,
+			"name": "Kona Ballroom 2"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 59
+	},
+	{
+		"short_abstract": "",
+		"speakers": [{
+			"id": 107,
+			"name": "Sean Rich",
+			"organisation": "Mozilla"
+		}],
+		"subtitle": null,
+		"language": "english",
+		"title": "IT: Document Vendor Procurement Process",
+		"track": {
+			"id": 3,
+			"title": "IT"
+		},
+		"start_time": "2016-12-07T19:00:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-07T21:30:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 13,
+			"name": "Kona Ballroom 4"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 63
+	},
+	{
+		"short_abstract": "",
+		"speakers": [{
+			"id": 15,
+			"name": "Jeff Bryner",
+			"organisation": "Mozilla"
+		}],
+
+		"subtitle": null,
+		"language": "english",
+		"title": "IT: Security Awareness 2017",
+		"track": {
+			"id": 3,
+			"title": "IT"
+		},
+		"start_time": "2016-12-07T19:00:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-07T20:00:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 13,
+			"name": "Kona Ballroom 4"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 65
+	},
+	{
+		"short_abstract": "",
+		"speakers": [],
+		"subtitle": null,
```

Comment:
speakers please. mahika many errors are repetitive, so please look for similar errors in your file please.